### ISSUE TITLE
Would this work for Tang Nano 9K FPGA?

### ISSUE THREAD

**Issue by louis030195** [vmunoz82/eda_tools#3]
I'm trying to do some dev with this FPGA but Gowin seems not to run on Mac, was wondering if I could use this Docker for it?

https://wiki.sipeed.com/hardware/en/tang/Tang-Nano-9K/Nano-9K.html

**Comment by vmunoz82** [comment]
Yes! it does work with Tang Nano 9K, I have myself that board too, but notice that Apicula is not at the level that the oficial Gowin tools, and sometimes the design synthesize, but don't work as expected, anyway Apicula gets nice improvements with the time.
